ansible/roles/jenkins-worker/templates/aix.rc2.j2

```diff
@@ -20,9 +20,7 @@ start )
               export HOME=/home/{{server_user}}; \
               export NODE_TEST_DIR=$HOME/tmp; \
               export NODE_COMMON_PIPE="$HOME/test.pipe"; \
-              export OSTYPE=AIX72; \
-              export GIT_SSL_CAINFO="$HOME/ca-bundle.crt"; \
-              export SSL_CERT_FILE="$HOME/ca-bundle.crt"; \
+              export OSTYPE=AIX7; \
```

Comment:
I'm not sure where the `OSTYPE` env var is used, do you? Maybe it should be left as-is in this PR, I see you've another PR to change "aix72" to "aix7", would this be better there?

Reply:
I got confused, its "rhel72" that is changing to "rhel7"... sorry! I'd be tempted to leave this single line alone, it does no harm, is possibly unused, and sometime we can track it down and figure out if it is dead code or not, but that's unrelated to the cert location fix, which is understood, and we can just do now in this PR.

Reply:
I only changed it because I thought it might have had something to do with select-compiler, but it doesnt Ill happily swap it back

Reply:
@sam-github @richardlau Im not 100% sure but there is a reference to `OSTYPE` in the makefile here https://github.com/nodejs/node/blob/024b6588313a0e32595317936ee279dec7a5a756/Makefile#L12

Could this be what its used for? and if so why are we setting it here when it gets set in the makefile?